packages/core/i18n/src/Translation.ts

```diff
@@ -1,10 +1,14 @@
 import { defineComponent, h } from 'vue'
-import type { VNodeChild, App } from 'vue'
+import type { VNodeChild, App, PropType } from 'vue'
 import type { IntlShapeEx } from './types'
 
-const TranslationComponent = (i18n: IntlShapeEx) => defineComponent({
+export const createI18nTComponent = (i18n: IntlShapeEx | null = null) => defineComponent({
```

Comment:
suggestion: Can we name this function `i18nTComponent` so that to use it, you can do this? 

```ts
const i18nT = i18nTComponent(i18nLocal)
```

Reply:
I think having two functions named `create...` might be confusing (I got asked 3 times yesterday how things worked and had to keep pointing them to the docs)

Reply:
done